app/javascript/components/supplemental_application/sections/preferences/AntiDisplacementHousingPanel.js

```diff
@@ -24,7 +24,7 @@ export const AntiDisplacementHousingPanel = ({ preferenceIndex, preference }) =>
             TODO: Add ability for users to change application member on pref.
             For now, we just show the current app member in a read-only field.
           */}
-          <input value={memberNameFromPref(preference)} disabled='true' />
```

Comment:
disabled='true' was giving me a console warning
disabled={true} was giving me a linting error on overcommit. 

I got this solution from https://stackoverflow.com/questions/38659884/using-boolean-value-of-attributes-in-jsx